typings/vscode-mssql.d.ts

```diff
@@ -114,6 +114,68 @@ declare module 'vscode-mssql' {
 		 * @returns A promise object for when the request receives a response
 		 */
 		sendRequest<P, R, E, R0>(requestType: RequestType<P, R, E, R0>, params?: P): Promise<R>;
+
+		/**
+		 * Get the server info for a connection
+		 * @param connectionInfo connection info of the connection
+		 * @returns server information
+	 	*/
+		getServerInfo(connectionInfo: IConnectionInfo): IServerInfo
+	}
+
+	/**
+	 * Information about a SQL Server instance.
+	 */
+	export interface IServerInfo {
```

Comment:
we now have two places defining the ServerInfo, should we remove the old one and use this instead?

Reply:
Yes, removed the old one.